cmd/nigiri/start.go

```diff
@@ -53,6 +53,8 @@ func startAction(ctx *cli.Context) error {
 		servicesToRun = append(servicesToRun, "lnd")
 		// Core Lightning Network
 		servicesToRun = append(servicesToRun, "lightningd")
+		// Sensei
+		servicesToRun = append(servicesToRun, "sensei")
```

Comment:
Is Sensei capable to work without the frontend ie. only API/RPC? If Yes, may be worth running it also in CI mode (ie. under the isCI branch) so the Nigiri Github Action can offer CI testing using sensei

https://github.com/vulpemventures/nigiri-github-action